app/services/hyrax/admin_set_service.rb

```diff
@@ -64,5 +66,17 @@ def data_attributes(admin_set)
         attrs['data-visibility'] = permission_template.visibility unless permission_template.visibility.blank?
         attrs
       end
+
+      # Count number of files from admin set works
+      # @param [Array] Solr search result
+      def count_files(results)
```

Comment:
This approach is only going to count files for the number of works returned.  This defaults to 10, right?